<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_TM_Custom_Msg</class>
 <widget class="QWidget" name="HNS_TM_Custom_Msg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="HNS_Message_Preview" name="GV_Custom_Preview1" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Custom_Preview2" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Custom_Preview3" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Custom_Preview4" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>370</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Change_Custom1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CHANGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Change_Custom2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CHANGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Change_Custom3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>520</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CHANGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Change_Custom4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>520</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CHANGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Back">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>710</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CUSTOM 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CUSTOM 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CUSTOM 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CUSTOM 4</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNS_Message_Preview</class>
   <extends>QWidget</extends>
   <header>hns_message_preview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
